Attempting to install parcel on node@18.0.0 under Windows fails

# 🐛 bug report

```
$ volta install node@18
$ mkdir ~/Desktop/testing_parcel
$ cd ~/Desktop/testing_parcel
$ npm install parcel
npm ERR! code 1
npm ERR! path C:\Users\ec\Desktop\testing_parcel\node_modules\lmdb
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node-gyp-build
npm ERR! Building the projects in this solution one at a time. To
        enable parallel build, please add the "-m" switch.
npm ERR!   lmdb-js.cpp
npm ERR! C:\Users\ec\AppData\Local\node-gyp\Cache\18.0.0\include\node\cppgc\common.h(9,10):
        fatal error C1083: Cannot open include file: 'v8config.h':
        No such file or directory [C:\Users\ec\Desktop\testing_parcel\node_modules\lmdb\build\lmdb.vcxproj]
...
```

## 🎛 Configuration (.babelrc, package.json, cli command)

None - this happens with a brand-new Node, and `npm install parcel` in an empty, new directory.

## 😯 Current Behavior

<details>

```
ec@Joyce-the-MSI-Creator-Z17 MINGW64 ~/Desktop/testing_parcel
$ npm install parcel
npm ERR! code 1
npm ERR! path C:\Users\ec\Desktop\testing_parcel\node_modules\lmdb
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c node-gyp-build
npm ERR! Building the projects in this solution one at a time. To enable parallel build, please add the "-m" switch.
npm ERR!   lmdb-js.cpp
npm ERR! C:\Users\ec\AppData\Local\node-gyp\Cache\18.0.0\include\node\cppgc\common.h(9,10): fatal error C1083: Cannot open include file: 'v8config.h': No such file or directory [C:\Users\ec\Desktop\testing_parcel\node_modules\lmdb\build\lmdb.vcxproj]
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.0.0
npm ERR! gyp info using node@18.0.0 | win32 | x64
npm ERR! gyp info find Python using Python version 3.10.4 found at "C:\Users\ec\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\python.exe"
npm ERR! gyp http GET https://nodejs.org/download/release/v18.0.0/node-v18.0.0-headers.tar.gz
npm ERR! gyp http 200 https://nodejs.org/download/release/v18.0.0/node-v18.0.0-headers.tar.gz
npm ERR! gyp http GET https://nodejs.org/download/release/v18.0.0/SHASUMS256.txt
npm ERR! gyp http GET https://nodejs.org/download/release/v18.0.0/win-x86/node.lib
npm ERR! gyp http GET https://nodejs.org/download/release/v18.0.0/win-x64/node.lib
npm ERR! gyp http GET https://nodejs.org/download/release/v18.0.0/win-arm64/node.lib
npm ERR! gyp http 200 https://nodejs.org/download/release/v18.0.0/SHASUMS256.txt
npm ERR! gyp http 404 https://nodejs.org/download/release/v18.0.0/win-x86/node.lib
npm ERR! gyp WARN ia32 node.lib was not found in https://nodejs.org/download/release/v18.0.0/win-x86/node.lib 
npm ERR! gyp http 404 https://nodejs.org/download/release/v18.0.0/win-arm64/node.lib
npm ERR! gyp http 200 https://nodejs.org/download/release/v18.0.0/win-x64/node.lib
npm ERR! gyp info find VS using VS2019 (16.11.32413.511) found at:
npm ERR! gyp info find VS "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools"
npm ERR! gyp info find VS run with --verbose for detailed information
npm ERR! gyp info spawn C:\Users\ec\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\python.exe
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   'C:\\Users\\ec\\AppData\\Local\\Volta\\tools\\image\\node\\18.0.0\\node_modules\\npm\\node_modules\\node-gyp\\gyp\\gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'msvs',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   'C:\\Users\\ec\\Desktop\\testing_parcel\\node_modules\\lmdb\\build\\config.gypi',   
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   'C:\\Users\\ec\\AppData\\Local\\Volta\\tools\\image\\node\\18.0.0\\node_modules\\npm\\node_modules\\node-gyp\\addon.gypi',
de_modules\\npm\\node_modules\\node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=C:\\\\Users\\\\ec\\\\AppData\\\\Local\\\\node-gyp\\\\Cache\\\\18.0.0\\\\<(target_arch)\\\\node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=C:\\Users\\ec\\Desktop\\testing_parcel\\node_modules\\lmdb',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'C:\\Users\\ec\\Desktop\\testing_parcel\\node_modules\\lmdb\\build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\MSBuild.exe
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   'build/binding.sln',
npm ERR! gyp info spawn args   '/clp:Verbosity=minimal',
npm ERR! gyp info spawn args   '/nologo',
npm ERR! gyp info spawn args   '/p:Configuration=Release;Platform=x64'
npm ERR! gyp info spawn args ]
npm ERR! gyp ERR! build error
npm ERR! gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\MSBuild\Current\Bin\MSBuild.exe` failed with exit code: 1
npm ERR! gyp ERR! stack     at ChildProcess.onExit (C:\Users\ec\AppData\Local\Volta\tools\image\node\18.0.0\node_modules\npm\node_modules\node-gyp\lib\build.js:194:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:527:28)
npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:291:12)
npm ERR! gyp ERR! System Windows_NT 10.0.22000
npm ERR! gyp ERR! command "C:\\Users\\ec\\AppData\\Local\\Volta\\tools\\image\\node\\18.0.0\\node.exe" "C:\\Users\\ec\\AppData\\Local\\Volta\\tools\\image\\node\\18.0.0\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd C:\Users\ec\Desktop\testing_parcel\node_modules\lmdb
npm ERR! gyp ERR! node -v v18.0.0
npm ERR! gyp ERR! node-gyp -v v9.0.0
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ec\AppData\Local\npm-cache\_logs\2022-04-22T04_33_50_000Z-debug-0.log

ec@Joyce-the-MSI-Creator-Z17 MINGW64 ~/Desktop/testing_parcel
$ node --version
v18.0.0

ec@Joyce-the-MSI-Creator-Z17 MINGW64 ~/Desktop/testing_parcel
$ volta install node@16
success: installed and set node@16.14.2 (with npm@8.5.0) as default                                                                                                                                                                      9

ec@Joyce-the-MSI-Creator-Z17 MINGW64 ~/Desktop/testing_parcel
$ node --version
v16.14.2

ec@Joyce-the-MSI-Creator-Z17 MINGW64 ~/Desktop/testing_parcel
$ npm --version
8.5.0

ec@Joyce-the-MSI-Creator-Z17 MINGW64 ~/Desktop/testing_parcel
$ npm install parcel

added 157 packages, and audited 158 packages in 6s

71 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities

ec@Joyce-the-MSI-Creator-Z17 MINGW64 ~/Desktop/testing_parcel
$ npx parcel --version
2.5.0
```

</details>

## 🌍 Your Environment

<!--- Include as many relevant details about the environment you experienced the bug in -->

| Software         | Version(s) |
| ---------------- | ---------- |
| Parcel           | 2.5.0
| Node             | 18.0.0
| npm/Yarn         | 8.6.0
| Operating System | Windows 11, Git Bash / MINGW64

--- a/packages/core/cache/package.json
+++ b/packages/core/cache/package.json
@@ -27,7 +27,7 @@
     "@parcel/fs": "2.5.0",
     "@parcel/logger": "2.5.0",
     "@parcel/utils": "2.5.0",
-    "lmdb": "2.2.4"
+    "lmdb": "2.3.5"
   },
   "peerDependencies": {
     "@parcel/core": "^2.5.0"

